vendor/zkllvm-metacraft-circuits/src/circuits_impl/verify_attestation_data_impl.h

```diff
@@ -183,65 +186,69 @@ VoteToken
     // attestation.
 
     base_field_type token = 0;
-
-    for (size_t i = 0; i < attestation.validators.size(); i++) {
-        auto& v = attestation.validators[i];
-        // Aggregate this validator's public key.
-        auto validator_pubkey = v.pubkey;
-        ///! pubkey_point = pubkey_to_G1(validator_pubkey)
-        ///! aggregated_point = bls.add(aggregated_point, pubkey_point)
-
-        // Check if this validator was part of the source state.
-        if (v.trusted) {
-            auto leaf = hash_validator(circuit_byte_utils::expand<64>(v.pubkey),
-                                       v.withdrawal_credentials,
-                                       v.effective_balance,
-                                       v.activation_eligibility_epoch,
-                                       v.activation_epoch,
-                                       v.exit_epoch,
-                                       v.withdrawable_epoch);
-            // Hash the validator data and make sure it's part of:
-            // validators_root -> state_root -> block_root.
-            ssz_verify_proof(
-                attestation.validators_root, leaf, v.validator_list_proof, 0x020000000000ul + v.validator_index);
-
-            // TODO: Needed?
-            // assert spec.is_active_validator(
-            //     v['activation_epoch'],
-            //     v['exit_epoch'],
-            //     EPOCH,
-            // )
-
-            // Include this validator's pubkey in the result.
-            base_field_type element;
-            memcpy(&element, &(v.pubkey), sizeof(element));
-            token = (token + (element * sigma));
+    for (size_t i = 0; i < std::remove_reference<decltype(attestation.validators)>::type::capacity; i++) {
```

Comment:
add `capacity` static constexpr function to `static_vector` and type `attestation.validators.capacity()`